### PR TITLE
Expand Pedia Quick Start Guide

### PR DESCRIPTION
**Pull request by st-pa-fo**
--> more elaborate noob-pedia-greetings (with minor typo and **major** style adjustments)

### CODE REVIEW

File: default/stringtables/en.txt [hunk 1]
```diff
@@ -6674,17 +6674,27 @@ GREETINGS_GUIDE_TEXT
 
 FreeOrion is a free, open source, turn-based space empire and galactic conquest (4X) computer game being designed and built by the FreeOrion project. FreeOrion is inspired by the tradition of the Master of Orion games, but is not a clone or remake of that series or any other game.
 
-Further information is available at freeorion.org
+Further information is available at http://www.freeorion.org
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]: revert. this is wrong two ways

File: default/stringtables/en.txt [hunk 2]
```diff
@@ -6674,17 +6674,27 @@ GREETINGS_GUIDE_TEXT
...
 '''
 
 QUICK_START_GUIDE_TITLE
 **Quick Start Guide**
 
 QUICK_START_GUIDE_TEXT
-'''This is a brief introduction to FreeOrion gameplay. A somewhat less brief Quick Play Guide is available at freeorion.org (current URL: <url  http://freeorion.org/index.php/V0.4_Quick_Play_Guide>freeorion.org/index.php/V0.4_Quick_Play_Guide</url> ).
+'''The Pedia is ment to assist with and inform about the many strange new features of this great game. Right-clicking on many items on the game screen will lead to specific Pedia articles. But first, lets have a look around.
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]: have another editing pass. this needs more work

Comment (st-pa-fo) [reply default/stringtables/en.txt]: is ok if i don't bother with the content checksums? lets that one test fail though.

Comment (geoffthemedio) [reply default/stringtables/en.txt]: it should be fixed when all the other changes are final... until then, it doesn't matter

File: default/stringtables/en.txt [hunk 3]
```diff
@@ -6681,10 +6681,20 @@ QUICK_START_GUIDE_TITLE
 **Quick Start Guide**
 
 QUICK_START_GUIDE_TEXT
-'''This is a brief introduction to FreeOrion gameplay. A somewhat less brief Quick Play Guide is available at freeorion.org (current URL: <url  http://freeorion.org/index.php/V0.4_Quick_Play_Guide>freeorion.org/index.php/V0.4_Quick_Play_Guide</url> ).
+'''The Pedia is ment to assist the player with and provide reference material about all aspects of this game, its mechanics and its various content. Right-clicking on many items on the game screen will lead to specific Pedia articles. And now, to get started.
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]:
```suggestion
'''This Pedia provides information about many aspects of FreeOrion, including mechanics and content, as well as in-game objects and empires during a game. The pedia can be accessed from the top level index page, which provides categorized lists of articles, by entering text in the box at the top of the pedia page to search articles for the entered text, or via right-click popup menu links from many in-game objects in the interface.
```

File: default/stringtables/en.txt [hunk 4]
```diff
@@ -6681,10 +6681,20 @@ QUICK_START_GUIDE_TITLE
...
 
-[[DETAILED_PEDIA_REFERENCE]]
-Important sections include [[encyclopedia CATEGORY_GAME_CONCEPTS]], [[encyclopedia CATEGORY_GUIDES]], and the [[encyclopedia ENC_METER_TYPE]] list.
+There are three urgent problems that need to be addressed.
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]:
```suggestion
At the start of a new game, there are three important issues that should be addressed:
```

File: default/stringtables/en.txt [hunk 5]
```diff
@@ -6681,10 +6681,20 @@ QUICK_START_GUIDE_TITLE
...
+
+1. All starships are initially idle in orbit around the central star of the home world's planetary system. Visible in the [[encyclopedia MAP_WINDOW_ARTICLE_TITLE]] is a star map, with the home system highlighted in the center. Next to it, a bit up and to the right, is the fleet. Ships can be selected by left-clicking on the fleet symbol and then given march orders by right-clicking a target star. If ships get no orders, they will stay idle. Scouts should be sent out to uncover more of the map. See also [[encyclopedia FLEET_MOVEMENT_ARTICLE_TITLE]].
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]:
```suggestion
1. Fleets are initially idle at homeworld's planetary system. Visible in the [[encyclopedia MAP_WINDOW_ARTICLE_TITLE]] is a star map, with the home system selected and highlighted in the center. Next to it, a bit up and to the right, is the fleets icon. Fleets can be selected by left-clicking on the fleet symbol on the map, which will open the fleets window, where all the fleets in a system and their contained ships can be viewed. If a fleet is selected, it can be given movement orders by right-clicking a nearby target star that is connected to the fleet's system by a starlane. Fleets that are not given move orders will stay idle. A good first step is often to send scout fleets to explore nearby systems, which will reveal more about their contents and any further starlane connections beyond them. See also [[encyclopedia FLEET_MOVEMENT_ARTICLE_TITLE]].
```

**Comment by st-pa-fo** [comment]
done and squashed.

File: default/stringtables/en.txt [hunk 6]
```diff
@@ -7215,7 +7225,9 @@ SITREP_SHOWALL_TEMPLATES
 Show all blocked SitReps
 
 SITREP_WELCOME
-[[encyclopedia GREETINGS_GUIDE_TITLE]] [[GREETINGS_GUIDE_REFERENCE]]
+'''[[encyclopedia GREETINGS_GUIDE_TITLE]] [[GREETINGS_INTRO]]
+If you are a newcomer to this universe, please click on the word "[[GREETINGS_GUIDE_TITLE]]" to help us get you started.
```
Comment (TheSilentOne1) [inline default/stringtables/en.txt]: I'd rephrase to: "... for some basic advice on how to play." (developers don't need mentioning)

Comment (st-pa-fo) [reply default/stringtables/en.txt]: done rephrasing. if there are no further suggestions, i would then proceed to squash this thing up again and make a short and concise commit message

Comment (geoffthemedio) [reply default/stringtables/en.txt]: the rest of the text I didn't make suggestions for also needs similar rewriting

Comment (st-pa-fo) [reply default/stringtables/en.txt]: oh kay. i'll have a fresh look at it next week.

File: default/stringtables/en.txt [hunk 7]
```diff
@@ -6681,10 +6681,20 @@ QUICK_START_GUIDE_TITLE
...
 
-[[DETAILED_PEDIA_REFERENCE]]
-Important sections include [[encyclopedia CATEGORY_GAME_CONCEPTS]], [[encyclopedia CATEGORY_GUIDES]], and the [[encyclopedia ENC_METER_TYPE]] list.
+At the start of a new game, there are three important issues that should be addressed:
+
+1. Fleets are initially idle at homeworld's planetary system. Visible in the [[encyclopedia MAP_WINDOW_ARTICLE_TITLE]] is a star map, with the home system selected and highlighted in the center. Next to it, a bit up and to the right, is the fleets icon. Fleets can be selected by left-clicking on the fleet symbol on the map, which will open the fleets window, where all the fleets in a system and their contained ships can be viewed. If a fleet is selected, it can be given movement orders by right-clicking a nearby target star that is connected to the fleet's system by a starlane. Fleets that are not given move orders will stay idle. A good first step is often to send scout fleets to explore nearby systems, which will reveal more about their contents and any further starlane connections beyond them. See also [[encyclopedia FLEET_MOVEMENT_ARTICLE_TITLE]].
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]:
```suggestion
1. Fleets are initially idle at homeworld's planetary system. Visible in the [[encyclopedia MAP_WINDOW_ARTICLE_TITLE]] is a star map, with the home system icon selected and highlighted in the center. At the top right of the system icon is a fleet icon. Fleets can be selected by left-clicking on the fleet icon on the map, which will open the fleets window, where all the fleets in a system and their contained ships can be viewed. If a fleet is selected, it can be given movement orders by right-clicking a nearby target star that is connected to the fleet's system by a starlane. Fleets that are not given move orders will stay idle. A good first step is often to send scout fleets to explore nearby systems, which will reveal more about their contents and any further starlane connections beyond them. See also [[encyclopedia FLEET_MOVEMENT_ARTICLE_TITLE]].
```

File: default/stringtables/en.txt [hunk 8]
```diff
@@ -6681,10 +6681,20 @@ QUICK_START_GUIDE_TITLE
...
+
+2. Industry on the home world is not producing anything until told what to manufacture next. A single click on the red cogwheel in the title bar or double-clicking the image of the home planet on the right side of the screen will open up the [[encyclopedia PRODUCTION_WINDOW_ARTICLE_TITLE]]. Scheduling of production runs is done by double-clicking entries from the list of "Producible Items". If they do not yet use up all the available production points, there will a red exclamation mark next to the cogwheel symbol. Return to the full map view by clicking on the cogwheel again or hitting escape (no need to worry, the production orders are not lost when hitting escape, that only returns to the map view.)
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]:
```suggestion
2. Industry on the home world is not producing anything until told what to manufacture next. A single click on the red cogwheel in the bar of icons and numbers at the top of the screen, or double-clicking the image of the home planet on the right side of the screen, will open up the [[encyclopedia PRODUCTION_WINDOW_ARTICLE_TITLE]]. Production can be ordered is done by double-clicking entries from the list of "Producible Items". If the production queue does not yet use up all the available production points for the current turn, there will a red exclamation mark next to the cogwheel symbol. A good first step is to order the production of more scout ships, or a military command building. After ordering production, the production view can be closed by clicking on the cogwheel again or hitting escape.
```

File: default/stringtables/en.txt [hunk 9]
```diff
@@ -6681,10 +6681,20 @@ QUICK_START_GUIDE_TITLE
...
+
+3. Scientific researchers must be told on which topics they should focus so as not to waste the abundance of creativity expressed by the number next to the blue Erlenmeyer flask in the title bar. Clicking on it will open the research window with the technology-tree, see also [[encyclopedia RESEARCH_TECH_GUIDE_TITLE]]. There are pedia-articles about every researchable topic to make the decision easier, what to research next. Double-clicking technologies will schedule them for research which uses up the available research points. When enough items are scheduled, the red exclamation sign next to the Erlenmeyer flask will disappear, indicating that there is no more excess of creativity. Click the Erlenmeyer flask or hit the Escape key to return to the map.
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]:
```suggestion
3. Research on the homeworld is not being directed effectively until research priorities are specified. A single click on the blue Erlenmeyer flask in the title bar will open the research window with the technology-tree. See also [[encyclopedia RESEARCH_TECH_GUIDE_TITLE]]. Double-clicking technologies will schedule them for research, which consumes the available research points. A good first step is often to research a technology that will give some bonus to the homeworld, such as [[tech GRO_PLANET_ECOL]] or [[tech PRO_ROBOTIC_PROD]].
```
